zeek/plugin/src/runtime-support.cc

```diff
@@ -165,6 +165,15 @@ hilti::rt::Bool rt::is_orig() {
     else
         throw ValueUnavailable("is_orig() not available in current context");
 }
+std::string rt::uid() {
+    auto cookie = static_cast<Cookie*>(hilti::rt::context::cookie());
+    assert(cookie);
```

Comment:
This seems pretty drastic and hard to work around. Could we instead either return an `optional<string>` here, or reuse below's `throw ValueUnavailable("uid() not available in current context");`?

Reply:
cookie is always available, it's set by the plugin (otherwise it's a bug, hence the assert)